### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by vikasz1**
Alka Please review my code and give your valuable feedback.

### CODE REVIEW

File: sample.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+print('hello world')
```
Comment (alkaz1) [inline sample.py]: This line is useless. Please remove it and merge this branch. Also delete the branch

Comment (vikasz1) [reply sample.py]: okay I'll do it. :)

Comment (vikasz1) [reply sample.py]: I have made necessary changes. Please review again.

**Review by alkaz1 — APPROVED** [review]
Looks nice to me.

**Pull request merged**